Runtime/ui/renderer/cmdbufferCanvas/rendering/canvas_clip.cs

```diff
@@ -35,7 +35,8 @@ class ClipElement : PoolObject {
             newElement.saveCount = saveCount;
 
             var pathCache = uiPath.flatten(scale);
-            var fillMesh = pathCache.getFillMesh(out newElement.convex);
+            pathCache.computeFillMesh(0.0f, out newElement.convex);
```

Comment:
Why computeFillMesh and computeStrokeMesh returns different values ( void vs. uiMeshMesh). They should behave similarly according to their names?